<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FontDialog</class>
 <widget class="QDialog" name="FontDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="doneBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="fontList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>171</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="fontSize">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>269</y>
     <width>60</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>32</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>29</y>
     <width>100</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Font Family</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>274</y>
     <width>100</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Font Size</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>271</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
